Polyhedron/demo/Polyhedron/Scene_surface_mesh_item.cpp

```diff
@@ -57,8 +57,8 @@ struct Scene_surface_mesh_item_priv{
   //! fill the flat data vectors.
   void
   triangulate_facet(face_descriptor fd,
-                    SMesh::Property_map<face_descriptor, Kernel::Vector_3 > *fnormals,
-                    SMesh::Property_map<face_descriptor, CGAL::Color> *fcolors,
+                    CGAL::Properties::Property_map<face_descriptor, Kernel::Vector_3 > *fnormals,
+                    CGAL::Properties::Property_map<face_descriptor, CGAL::Color> *fcolors,
```

Comment:
@sgiraudot Note for another time: that kind of required change in the code of the demo was an indication that the API was broken by the PR.

Cross-reference: PR #1890 is dedicated to fixing this, and restore the proper API.